js/crm.ajax.js

```diff
@@ -25,10 +25,10 @@
     var url,
       frag = path.split('?');
     // Encode url path only if slashes in placeholder were also encoded
-    if (tplURL[mode].indexOf('civicrm/placeholder-url-path') >= 0) {
-      url = tplURL[mode].replace('civicrm/placeholder-url-path', frag[0]);
+    if (tplURL[mode].indexOf(CRM.config.basePage + '/placeholder-url-path') >= 0) {
+      url = tplURL[mode].replace(CRM.config.basePage + '/placeholder-url-path', frag[0]);
     } else {
-      url = tplURL[mode].replace('civicrm%2Fplaceholder-url-path', encodeURIComponent(frag[0]));
+      url = tplURL[mode].replace(CRM.config.basePage + '%2Fplaceholder-url-path', encodeURIComponent(frag[0]));
```

Comment:
I don't get it. If `basePage` is only used on the front-end, then why is it being used unconditionally on every page? I don't see any place that switches between `basePage` vs `'civicrm'` depending on `mode`.